apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,71 +17,51 @@ import * as AllExamples from 'examples';
+An alert is an element that notifies the user of something important that is not user initiated. If you need to alert the user because of something they did, consider using a [toast](toast) notification or an inline message. Instead, an alert is used to notify the user of something unrelated to what they just did. Examples of this might be announcing a new feature within the product, or alerting the user of scheduled server maintenance and downtime.
 
-## Appearance
+## Usage
 
-The alert message must be concise, we recommend no more than 256 characters. If you feel the need to explain in further detail, you may use an optional hyperlink at the end of the alert message to explain more on a new page. You may also include an optional close icon (close icon is required for sticky alerts).
+The alert message must be concise, we recommend no more than 256 characters. If you feel the need to explain in further detail, you may use an optional hyperlink at the end of the alert message to explain more on a new page. You may also include an optional close icon (close icon is required for [sticky](#sticky) alerts).
 
-### Informational
+### Composition API
 
-Default style when there is no defined status for the alert. Inform users about events that they should be aware of, but that are not disruptive to their work.
+You can build Alerts with subcomponents that are fully customizable.
 
-<LiveExample src='Alert.informational.tsx'>
-  <AllExamples.AlertInformationalExample client:load />
-</LiveExample>
-
-### Positive
-
-The positive alert is used to inform the user of a positive status.
-
-<LiveExample src='Alert.positive.tsx'>
-  <AllExamples.AlertPositiveExample client:load />
-</LiveExample>
-
-### Warning
+We provide:
 
-The warning alert is used to inform the user of something that is not currently disruptive to their work, but may be soon or eventually.
+- Alert.Wrapper
+- Alert.Icon
+- Alert.Message
+- Alert.Action
+- Alert.CloseButton
 
-<LiveExample src='Alert.warning.tsx'>
-  <AllExamples.AlertWarningExample client:load />
-</LiveExample>
-
-### Negative
+The type of alert can be specified by utilizing the `type` prop inside `<Alert.Wrapper>`. There are four distinct types of alerts that can be chosen from, namely positive, negative, warning, or informational. Positive alerts are an indication that a task or action has been successfully completed, while warning alerts require attention but do not result in any immediate disruption. Negative alerts, however, signify system outages or failures. Informational alerts are intended to provide users with updates on non-disruptive tasks or work.
```

Comment:
This paragraph seems to long and hard to read. Can we structure it better. Maybe bullet points for each status? Or maybe let's make the `"<status> alerts"` (e.g. `"Position alerts"`) bold? That way we can easily find info about the status we are looking for.